features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -190,6 +199,23 @@ class Configuration extends React.Component {
             return apiTransport.includes(type);
         }
     }
+    setSecurityScheme = (updatedSecurityScheme) => {
+        this.setState((oldState) => {
+            let { securityScheme } = oldState;
+            securityScheme = updatedSecurityScheme;
+            return { securityScheme };
+        });
+    }
+    addToArray(element, array) {
+        if (!array.includes(element)) {
+            array.push(element);
```

Comment:
can't we move this method in to  APISecurity component?

Reply:
it is used within configuration class as well

Reply:
seems like it is only been used to be passed down as props. So I dont think we need to define it here.

Reply:
You are right. However, the 'removeFromArray' method is used in both places, it would be inconsistent if we move only one of the methods to APISecurity component.

Reply:
noted